bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/robot/ProblemRobot.scala

```diff
@@ -459,10 +459,20 @@ object TimeProblems {
       }
 
   val SCHEDULING_SECTION = "Time Requests"
-}
 
-import ProblemRobot._
-import TimeProblems._
+  // REL-2032 Check that none of the requested times per partner are zero
+  def partnerZeroTimeRequest(p: Proposal, s:ShellAdvisor): List[ProblemRobot.Problem] = p.proposalClass match {
+    case g:GeminiNormalProposalClass =>
+      val probs = for {
+          sub <- g.subs.swap.right
+        } yield for {
```

Comment:
Can someone explain to me why you would do a:
`for { ... } yield for { ... } yield ...`?
I'm not sure what this does. Is it equivalent to a nested loop in this case so the result is an iterable of iterable of Problem?